message_test.go

```diff
@@ -126,6 +119,17 @@ func TestNewMessage(t *testing.T) {
 	}
 }
 
+func TestNewMessage(t *testing.T) {
+	SetServerResponse(200, messageObject)
```

Comment:
Sometimes constants like `http.StatusOK` are more readable. If you like, you can use them.